packages/react/src/components/Authenticator/ResetPassword/ResetPassword.tsx

```diff
@@ -29,6 +29,8 @@ export const ResetPassword = (): JSX.Element => {
     });
   };
 
+  const inputLabel = I18n.get('Enter your username');
```

Comment:
Should this have a `// TODO` to indicate we want this to say `Enter your email` or `Enter your username` or `Enter your phone number` based on the primary login mechanism?

Reply:
Eh. It'll come up in the QA session, right?